cpp/src/io/json/read_json.cu

```diff
@@ -441,4 +386,101 @@ table_with_metadata read_json(host_span<std::unique_ptr<datasource>> sources,
                              {partial_tables[0].metadata.schema_info}};
 }
 
+}  // anonymous namespace
+
+device_span<char> ingest_raw_input(device_span<char> buffer,
+                                   host_span<std::unique_ptr<datasource>> sources,
+                                   compression_type compression,
```

Comment:
This parameter is now unused (as it should be)

Reply:
Removed.